Trimble.Modus.Components/Controls/TMModal/TMModalContents.xaml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<popup:PopupPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                 x:ClassModifier="internal"
+                 x:Class="Trimble.Modus.Components.Modal.TMModalContents"
+                 xmlns:color="clr-namespace:Trimble.Modus.Components.Styles"
+                 xmlns:popup="clr-namespace:Trimble.Modus.Components.Popup.Pages"
+                 xmlns:converters="clr-namespace:Trimble.Modus.Components.Converters"
+                 xmlns:constants="clr-namespace:Trimble.Modus.Components.Constant"
+                 Title="TMModalContents"
+                 BackgroundColor="{x:StaticResource ModalGraySemiTransparent}">
+  <ContentPage.Resources>
+    <ResourceDictionary>
+      <ResourceDictionary.MergedDictionaries>
+        <color:Colors />
+      </ResourceDictionary.MergedDictionaries>
+      <converters:ImageNullOrEmptyBoolConverter x:Key="ImageNullOrEmptyBoolConverter" />
+      <converters:StringNotNullOrEmptyBoolConverter x:Key="StringNotNullOrEmptyBoolConverter" />
+    </ResourceDictionary>
+  </ContentPage.Resources>
+  <!-- Note: Using this parent container to create shadow effect for the modal since we are unable to add shadow to the modal/ModalContainer directly.
+             Change the background color of the parent container to change the color of the shadow.
+             Change the padding of the parent container to change the size of the shadow -->
+  <Border x:Name="ParentContainer"
+          StrokeThickness="0"
+          Padding="3"
+          BackgroundColor="{x:StaticResource ModalShadowColor}"
+          VerticalOptions="Center"
+          HorizontalOptions="Center"
+          StrokeShape="RoundRectangle 5">
+    <Border x:Name="ModalContainer"
+            Padding="16"
+            VerticalOptions="Center"
+            HorizontalOptions="Fill"
+            StrokeShape="RoundRectangle 4"
+            Stroke="{x:StaticResource Black}"
+            StrokeThickness="0"
+            BackgroundColor="White">
+      <Border.Shadow>
+        <Shadow Brush="{x:StaticResource Black}"
+                Offset="0,0"
+                Opacity="0.2"
+                Radius="8" />
+      </Border.Shadow>
+      <Grid RowSpacing="16">
+        <Grid.ColumnDefinitions>
```

Comment:
We can remove this column definition since its required for header alone we can handle this inside the stacklayout itself